### ISSUE TITLE
Test drive module

### ISSUE THREAD

**Issue by regisphilibert** [theNewDynamic/hugo-module-tnd-seo#2]
- https://github.com/theNewDynamic/aaldef.org/pull/74
- https://github.com/theNewDynamic/thenewdynamic.com/pull/114

@budparr I implemented the WIP new SEO module on tnd.com and aaldef.org.

I get the average same build time, on both my local and netlify, but feel free to take it for a spin on your computer.

PS: --templateMetrics reports a build 2.5 time slower, but final build time does not reflect that 🤷‍♂️

**Comment by budparr** [comment]
> a build 2.5 time slower

Is that like 250%?

**Comment by regisphilibert** [comment]
yes but even so, the build time is equivalent. I'm not sure I'm reading --templateMetrics well. 
The implementation for tnd.com uses the workshop content files to boost speed.

Now I realize you can only compare AALDEF because tnd.com does not run the seo old component on main. Plus on the special branch, I import the worshop's 1000's content file for stress testing it.

On my computer TND.com:
Main with no SEO module and not the 1000s content files runs a build of ~600ms
The SEO branch with the module and the 1000s files amount to ~1200ms.
So the Module with 1000 content files only adds less than a second, in spite of the --templateMetrics numbers...

We can chat if you want.
